corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -364,6 +364,18 @@ <h4 class="panel-title panel-title-nolink">{% trans "Search and Claim Options" %
                          data-bind="value: custom_related_case_property"/>
               </div>
           </div>
+          <div class="form-group">
+            <label class="control-label {% css_label_class %}">
+              {% trans "Include all related cases in search results" %}
+            </label>
+            <div>
+              <div class="{% css_field_class %} checkbox">
+                <label>
+                  <input type="checkbox" data-bind="checked: include_all_related_cases"/>
+                </label>
+              </div>
+            </div>
+          </div>
```

Comment:
Is this being bundled under `USH_CASE_CLAIM_UPDATES`?

Reply:
Yes it is, this change will be under USH_CASE_CLAIM_UPDATES FF